### PR TITLE
UI : Make the Activity card popup style similar to explore page

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- Closes : #8098
- Make the Activity card popup style similar to explore page

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
Old : 
image

New :
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx [hunk 1]
```diff
@@ -161,65 +162,65 @@ const EntityPopOverCard: FC<Props> = ({ children, entityType, entityFQN }) => {
...
+                  width="20"
+                />
+                <Typography.Text className="text-xs">
+                  {getEntityName(entityData.owner)}
+                </Typography.Text>
+              </Space>
             ) : (
-              <span className="tw-text-grey-muted">No Tier</span>
+              <Typography.Text className="text-xs text-grey-muted">
+                No Owner
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx:182]: localisation

File: openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx [hunk 2]
```diff
@@ -161,65 +162,65 @@ const EntityPopOverCard: FC<Props> = ({ children, entityType, entityFQN }) => {
...
-        </div>
-
-        <div
-          className="description-text tw-mt-1"
-          data-testid="description-text">
+          <span className="text-grey-muted">|</span>
+          <Typography.Text
+            className="text-xs text-grey-muted"
+            data-testid="tier">
+            {entityTier ? entityTier : ' No Tier'}
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx:190]: localisation

File: openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx [hunk 3]
```diff
@@ -161,65 +162,65 @@ const EntityPopOverCard: FC<Props> = ({ children, entityType, entityFQN }) => {
...
+        <div className="description-text m-t-sm" data-testid="description-text">
           {entityData.description ? (
             <RichTextEditorPreviewer
               enableSeeMoreVariant={false}
               markdown={entityData.description}
             />
           ) : (
-            <span className="tw-no-description">No description</span>
+            <Typography.Text className="text-xs text-grey-muted">
+              No description
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx:202]: localisation

**Review by chirag-madlani — CHANGES_REQUESTED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx (hunk 1 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx]: @chirag-madlani updated!

File: openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx (hunk 2 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx]: @chirag-madlani updated!

File: openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx (hunk 3 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx]: @chirag-madlani updated!

File: openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx [hunk 4]
```diff
@@ -179,15 +181,21 @@ const EntityPopOverCard: FC<Props> = ({ children, entityType, entityFQN }) => {
               </Space>
             ) : (
               <Typography.Text className="text-xs text-grey-muted">
-                No Owner
+                {t('label.no-entity', {
+                  entity: 'Owner',
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx:185]:
```suggestion
                  entity: t('label.owner'),
```

File: openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx [hunk 5]
```diff
@@ -179,15 +181,21 @@ const EntityPopOverCard: FC<Props> = ({ children, entityType, entityFQN }) => {
...
           </div>
           <span className="text-grey-muted">|</span>
           <Typography.Text
             className="text-xs text-grey-muted"
             data-testid="tier">
-            {entityTier ? entityTier : ' No Tier'}
+            {entityTier
+              ? entityTier
+              : t('label.no-entity', {
+                  entity: 'Tier',
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx:197]:
```suggestion
                  entity: t('label.tier'),
```

File: openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx [hunk 6]
```diff
@@ -199,7 +207,9 @@ const EntityPopOverCard: FC<Props> = ({ children, entityType, entityFQN }) => {
             />
           ) : (
             <Typography.Text className="text-xs text-grey-muted">
-              No description
+              {t('label.no-entity', {
+                entity: 'Description',
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx:211]:
```suggestion
                entity: t('label.description'),
```

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]

**Comment by Ashish8689** [comment]
@Sachin-chaurasiya  changes done as per comments

**Review by Sachin-chaurasiya — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx [hunk 7]
```diff
@@ -161,65 +164,73 @@ const EntityPopOverCard: FC<Props> = ({ children, entityType, entityFQN }) => {
...
-          </div>
-          <span className="tw-mx-1.5 tw-inline-block tw-text-gray-400">|</span>
-          <div data-testid="tier">
-            {entityTier ? (
-              entityTier
+            {entityData.owner ? (
+              <Space align="center" size="small">
+                <ProfilePicture
+                  displayName={getEntityName(entityData.owner)}
+                  id=""
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx:174]: provide id or remove the field

File: openmetadata-ui/src/main/resources/ui/src/styles/app.less [hunk 8]
```diff
@@ -113,6 +113,114 @@
...
+  width: 6rem;
+}
+.w-32 {
+  width: 8rem;
+}
+.w-36 {
+  width: 9rem;
+}
+.w-72 {
+  width: 288px;
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/styles/app.less:142]: aren't we already having this class?

File: openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx (hunk 7 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx]: @ShaileshParmar11  updated!

File: openmetadata-ui/src/main/resources/ui/src/styles/app.less (hunk 8 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/styles/app.less:142]: @ShaileshParmar11  we are having, feel like due to the main merge and change in the position they are showing.

**Review by ShaileshParmar11 — DISMISSED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx [hunk 9]
```diff
@@ -137,7 +140,7 @@ const EntityPopOverCard: FC<Props> = ({ children, entityType, entityFQN }) => {
     return (
       <Link data-testid="entitylink" to={getEntityLink(entityType, entityFQN)}>
         <button
-          className="tw-text-info"
+          className="text-primary"
           disabled={AppState.isTourOpen}
           onClick={(e) => e.stopPropagation()}>
           <span>{entityFQN}</span>
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx:146]: Use antd button

File: openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx [hunk 10]
```diff
@@ -161,65 +164,73 @@ const EntityPopOverCard: FC<Props> = ({ children, entityType, entityFQN }) => {
...
-            className="tw-mt-2 tw-flex tw-flex-wrap tw-gap-1"
-            data-testid="tags">
-            {entityTags.map((tag) => (
-              <span
-                className="tw-border tw-border-main tw-px-1 tw-rounded tw-text-xs"
-                key={uniqueId()}>
-                {tag}
+          <>
+            <Divider className="m-b-xs m-t-sm" />
+            <div className="flex flex-start">
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx:220]: Use antd space here.

File: openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx [hunk 11]
```diff
@@ -161,65 +164,73 @@ const EntityPopOverCard: FC<Props> = ({ children, entityType, entityFQN }) => {
     }, []);
 
     return (
-      <div className="tw-w-80">
-        <div className="tw-flex">
+      <div className="w-500">
+        <Space align="center" size="small">
           <div data-testid="owner">
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx:169]: Can we give testid to the space component and remove this dive?

**Review by Sachin-chaurasiya — COMMENTED** [review]
@Ashish8689, Let's remove all the tailwind dependencies.

File: openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx (hunk 11 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx:169]: @Sachin-chaurasiya `div` is the child of the Space element which represents the owner. so we can't assign the testid to Space and even `div` is the parent of 2 condition child. so we can't remove that too.

File: openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx (hunk 9 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/common/PopOverCard/EntityPopOverCard.tsx]: @Sachin-chaurasiya updated!

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**